packages/components/addon/components/hds/app-footer/index.hbs

```diff
@@ -0,0 +1,23 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class="hds-app-footer" ...attributes>
+  {{!-- {{yield to="topContent"}} --}}
+  {{yield (hash ExtraContent=(component "hds/app-footer/extra-content"))}}
+
+  <ul class="hds-app-footer__list">
+    {{yield
+      (hash
+        Item=(component "hds/app-footer/item")
+        Link=(component "hds/app-footer/link")
+        StatusLink=(component "hds/app-footer/status-link")
```

Comment:
Looking at how they've decided to stack the blocks, I think another parent container is needed, and the `StatusLink` needs to go outside this list):

<img width="559" alt="screenshot_3101" src="https://github.com/hashicorp/design-system/assets/686239/6b7d8144-8cd2-4856-8667-aeaacb301ab1">

(in fact they also had an extra container, in their schema)